.travis.yml

```diff
@@ -11,43 +11,50 @@ before_install:
   - sudo apt install -y libtool libtool-bin automake bison libglib2.0
 
 # TODO: Look into splitting off some builds using a build matrix.
+# TODO: Move this all into a bash script so we don't need to write bash in yaml.
```

Comment:
+million

Reply:
"a million lines" you mean?

Reply:
Sorry, I meant a very exaggerated "+1", agreed with the TODO